glide.go

```diff
@@ -596,6 +596,44 @@ Example:
 			},
 		},
 		{
+			Name:  "info",
+			Usage: "Info prints some information about this project based on glide.yaml",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name: "format, f",
+					Usage: `Format of the information wanted. Variables:
+                        %n - name
+                        %d - description
+                        %h - homepage
+                        %l - license
+                        %v - version
+
+                        Examples:
+                        Given the project with the following glide.yaml:
+                        package: foo
+                        version: 1.0
+                        homepage: https://foo.io
+                        license: MIT
+                        description: Some foo description
+
+                        Then:
+                        glide info -f %n
+                              prints 'foo'
+                        glide info -f "Version: %v"
+                              prints 'Version: 1.0'
+                        glide info -f "%n - %v - %d - %h - %l"
+                              prints 'foo - 1.0 - Some foo description - https://foo.io - MIT'`,
+				},
+			},
+			Action: func(c *cli.Context) {
+				if c.IsSet("format") {
+					action.Info(c.String("format"))
```

Comment:
If no format is passed in then the help command is showed. Format is a mandatory option.